src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -191,6 +191,7 @@ public static partial class Convert
         public static int ToBase64CharArray(byte[] inArray, int offsetIn, int length, char[] outArray, int offsetOut, Base64FormattingOptions options) { throw null; }
         public static string ToBase64String(byte[] inArray, Base64FormattingOptions options) { throw null; }
         public static string ToBase64String(byte[] inArray, int offset, int length, Base64FormattingOptions options) { throw null; }
+        public static string ToBase64String(System.ReadOnlySpan<byte> bytes, Base64FormattingOptions options = Base64FormattingOptions.None) { throw null; }
```

Comment:
Is this done in corert as well? If not then it will fail on uap and uapaot builds.

Reply:
Convert.cs is in the shared partition.  Once it's merged into coreclr, it'll be mirrored over to corert.

Reply:
I see, that's great! I missed that part on your coreclr PR :)